Robot2019/src/main/java/frc/robot/lib/logging/Log.java

```diff
@@ -318,28 +318,28 @@ public static void putDataToSmartDashboard() {
     }
 
     /**
-     * @return How often data is logged
-     * @see #setDataLogFrequency(int)
+     * @return How often data is logged where each interval unit represents a data logging interval of ~20ms
```

Comment:
The period associated with a TimedRobot is configurable so it isn't necessarily anywhere near 20ms. That's just the default. the units of the interval are calls to logData(). Providing an example would be helpful. The example could be just in the logData() javadoc.